.github/workflows/deploy-docs.yaml

```diff
@@ -16,6 +16,11 @@ jobs:
       - uses: webfactory/ssh-agent@v0.5.0
         with:
           ssh-private-key: ${{ secrets.PROFILES_DOCS_WEAVEWORKS_DOCS_BOT_DEPLOY_KEY }}
+      - name: Fetch latest pctl binary
```

Comment:
While I understand why these are separate, this adds two downloads. Does it make sense to keep these as completely separate actions, not just like one action with two steps?

Reply:
not sure i get what you mean.

do you mean:
- combine deploy and build actions

or
- merge the fetch bin and deploy steps within this action?

Reply:
Yeah, I meant combine deploy and build. But maybe that would make that action too big?

Reply:
deploy should only run on main merge, whereas the build should happen on PRs to check for broken links etc.

this used to be done implicitly as the `docs` make command was added to `test`, but i split it out for clarity

Reply:
Ah okay, gotcha.

Reply:
I don't actually know if the deploy action will work yet

yolo